README.md

```diff
@@ -68,6 +71,7 @@ After writing this method, our new result should be:
 ```
 
 **think**: Where should we call `strip_punctuation`? Does it matter? Why?
+before we iterate and analyze the data. It does matter otherwise you will get bad data 
```

Comment:
I would say inside because the mood cannot be properly analyzed unless the punctuation is stripped from it.